src/shared/lib/PasswordValidation.ts

```diff
@@ -0,0 +1,94 @@
+const sixCharactersRegex = /^.{6,}$/;
+const lessThan72CharactersRegex = /^.{0,72}$/;
+const upperAndLowercaseRegex = /[a-z].*[A-Z]|[A-Z].*[a-z]/;
+const symbolOrNumberRegex = /[^a-zA-Z]/;
+
+export const atLeastSixText = 'At least 6 characters';
```

Comment:
A minor change request: 

In TS we generally use string enums (https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) to group together strings that can be referred to each other

for example this could become something like

```ts
export enum PasswordValidationText {
  AT_LEAST_6 = 'At lease 6 characters',
  UP_TO_72 = 'Up to 72 characters',
  ... etc
}
```

and then to use them you just need to import the enum, rather than all the text:
```ts
import { PasswordValidationText } from '@/shared/lib/PasswordValidation'

// to use just use dot syntax
console.log(PasswordValidationText.AT_LEAST_6)
```

This saves on having to import all the `const`s individually, and makes it easier to manage the strings too.

Reply:
A good example would be to see how we do errors: https://github.com/guardian/gateway/blob/main/src/shared/model/Errors.ts

Reply:
I feel like it would be a good fit here, as it saves on having to import all the strings individually in other places

Reply:
👌  https://github.com/guardian/gateway/pull/52/commits/c7902459507318e57de7b067ba46c6865d82553a